content/content-strategy/manage-content-requests/sign-off-content/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Sign off content
+description: Create a brief or checklist for final sign off, so the approver knows what they need to do.
+weight: 70
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md
+  - toc.md
+  - cards.md
+  - text.md
+  - fact.md #fact
+  - who-head.md
+  - who.md
```

Comment:
Please add the global feedback partial, `- /_shared/feedback.md` at the end of the `main` section

Reply:
Trisha: done